### PR TITLE
Add navigation bar

### PR DESCRIPTION
**Pull request by Zel-hub7**
1. We added the navigation bar as requried in the project.
2. We made sure that when you click on any of the links, the main area changes to display only one of the three content sections.
3. We used plain js.
4. We created three navigations and used display or hide on our project.

### CODE REVIEW

**Review by shella12 — COMMENTED** [review]
Hi @team, 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README.md. Please remove template logo (Microverse logo) from your README. Optionally, you can replace with your own logo if you have one. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by DammyShittu — COMMENTED** [review]
Hi @Zel-hub7, @GhostEsso and @Lawmsangi,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### The Key and Future Features sections have no content. Kindly add content to those sections.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -84,3 +84,28 @@ addBookForm.addEventListener('submit', (event) => {
...
+  allBooksContainer.style.display = 'block'; // Show all books container
+  addBookContainer.style.display = 'none'; // Hide add book container
+  about.style.display = 'none';
+});
+
+addBookLink.addEventListener('click', () => {
+  allBooksContainer.style.display = 'none'; // Hide all books container
+  addBookContainer.style.display = 'block'; // Show add book container
+  about.style.display = 'none';
+});
```
Comment (elisha2kyakpo1) [inline script.js]: - [ ] Great job adding the navigation links, but to make sure the app matches the requirements, only one page should be shown per each link. The first (home) page should only show the list of books (if any), if no book is added, the page should be blank.

pages

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Zel-hub7, @GhostEsso, @Lawmsangi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
👍 Nice UI.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @Zel-hub7, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested changes implemented ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**